lib/perl/Genome/Model/Tools/Vcf/EvaluateVcfs.pm

```diff
@@ -5,6 +5,7 @@ use warnings;
 
 use Cwd;
 use Path::Class;
+use JSON::XS;
```

Comment:
If you `use JSON;` it will use the XS version when it is available, but also fall back on the pure-Perl implementation otherwise.

From [the JSON::XS docs](http://search.cpan.org/~mlehmann/JSON-XS-3.01/XS.pm#DESCRIPTION):
>Beginning with version 2.0 of the JSON module, when both JSON and JSON::XS are installed, then JSON will fall back on JSON::XS (this can be overridden) with no overhead due to emulation (by inheriting constructor and methods). If JSON::XS is not available, it will fall back to the compatible JSON::PP module as backend, so using JSON instead of JSON::XS gives you a portable JSON API that can be fast when you need and doesn't require a C compiler when that is a problem.